docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md

```diff
@@ -100,6 +100,14 @@ editor
 viewer
 ```
 
+4. Add the _client roles_ mapper to the dedicated client scope with
```

Comment:
I think we should also add that it's required since Keycloak v25. Wdyt @chri2547?

Reply:
@mgyongyosi , I defer to your technical expertise here.